packages/lesswrong/server/migrations/20250508T230604.fixSuggestForAlignmentCommentsIndex.ts

```diff
@@ -0,0 +1,5 @@
+export const up = async ({db}: MigrationContext) => {
+}
```

Comment:
I think this still needs an `updateIndexes` call on `Comments`?

Reply:
D'oh, yes, that was supposed to be in this migration. Added.